test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs

```diff
@@ -30,6 +28,9 @@ protected override void GetLibrariesToPreload(List<string> libraries)
         {
             libraries.Add("DSCPython.dll");
             libraries.Add("DSIronPython.dll");
+            // Make sure Python Engine Selector Singleton has all the info up-to-date.
+            // This is not needed for Dynamo in normal use case but useful in unit test case.
+            PythonEngineSelector.Instance.ScanPythonEngines();
```

Comment:
will this need to be done when a user installs the ironPython package?

Reply:
@mjkkirschner Thinking about it.. Yes, regardless of dynamic loading or not, seems need to be called.